RELEASES.md

```diff
@@ -271,6 +272,13 @@ may be completely refactored or have breaking changes in _minor_ releases.
 
 We will try not break the tool in _patch_ releases.
 
+### Daemon Configuration
+
+The daemon's configuration file, commonly located in `/etc/containerd/config.toml`
+is versioned and backwards compatible.  The `version = 2` field in the config
+file specifies the config's version.  If no version number is specified inside
+the config file then it is assumed to be a version 1 config and parsed as so.
```

Comment:
minor: "parsed as such." at the end?

Reply:
they both sound correct to me but i'm from the middle of nowhere